### ISSUE TITLE
How did Adam & Eve end up making Chinese babbies?

### ISSUE THREAD

**Issue by gctommy** [god/man#5]
Why god, why ah?

**Comment by Chatto** [comment]
Yeah, I feel this 100% in the identity theft!!! Hot and horny Bolster, make it HONG, Bolster!

**Comment by Chatto** [comment]
See I have this issue where Black kids in america get handed money, look healthy and well dressed, but are unaware that this is not for them...

**Comment by Chatto** [comment]
And yes there are Black kids who can speak better Chinese than me...usually cuz they had an opportunity to learn more than me

**Comment by Chatto** [comment]
My Angry Tamagotchi thinks the Secret Chicago Public Zool system is core/udp.

**Comment by Chatto** [comment]
<br> and <br/> both speak Spamish , right?

**Comment by Chatto** [comment]
rs-anime, risingtideonline, kubikdragon, fishbox, vgdesk...any questions? faq yu
